src/AI/WallyAI.java

```diff
@@ -6,21 +6,46 @@
 import CommandingOfficers.Commander;
 import CommandingOfficers.CommanderAbility;
 import Engine.*;
+import Engine.GamePath.PathNode;
+import Engine.Utils.SearchNode;
 import Engine.Combat.BattleSummary;
 import Engine.Combat.CombatEngine;
+import Engine.Combat.StrikeParams;
 import Engine.Combat.CombatContext.CalcType;
-import Engine.UnitActionLifecycles.BattleLifecycle;
+import Engine.UnitActionLifecycles.CaptureLifecycle;
+import Engine.UnitActionLifecycles.BattleLifecycle.BattleAction;
+import Engine.UnitActionLifecycles.CaptureLifecycle.CaptureAction;
 import Engine.UnitActionLifecycles.WaitLifecycle;
 import Terrain.*;
 import Units.*;
 import Units.MoveTypes.MoveType;
 
-/**
- *  Wally values units based on firepower and the area they can threaten.
- *  He tries to keep units safe by keeping them out of range, but will also meatshield to protect more valuable units.
- */
```

Comment:
Oh, good. I was worried about having any kind of description here would make the file too long.

Reply:
It was duplicated with the user-facing one.